numpy/add_newdocs.py

```diff
@@ -663,14 +663,18 @@ def luf(lamdaexpr, *args, **kwargs):
         will only be made if __array__ returns a copy, if obj is a
         nested sequence, or if a copy is needed to satisfy any of the other
         requirements (`dtype`, `order`, etc.).
-    order : {'C', 'F', 'A'}, optional
+    order : {'C', 'F', 'A', 'K'}, optional, default 'K'
```

Comment:
Default value should not be mentioned on this line, althogh it is customary to put it first in the list of alternatives. Maybe `Specify the order of the array, default is 'K'`. For the rest, it would might be better to list each option in turn, or maybe a table with `copy` value along horizontal axis and `order` value vertically. Something like
```
    ===== =========  ==========
    order   copy=F     copy=T
    ===== =========  ==========
    'K'   Arbitrary  Near input
    'A'   Arbitrary  C order
    'C'   C order    C order
    'F'   F order    F order
    ===== =========  ==========
```
@seberg, can you check for correctness of the above?